docs/docs/naming.md

```diff
@@ -44,9 +74,9 @@ $ dig _search._tcp.marathon.mesos SRV
 
 ;; ANSWER SECTION:
 _search._tcp.marathon.mesos.	60 IN SRV 0 0 31302 10.254.132.41.
-``` 
+```
 
-SRV records are generated only for tasks that have been allocated a specific port through Mesos. 
+*Note*: SRV records are generated only for tasks that have been allocated a specific port through Mesos and that have no container IP assigned.
```

Comment:
my current thinking is that if DiscoveryInfo is specified then we generate SRV's:

- `_my-task._tcp.my-framework.mesos` --> `my-task.my-framework.mesos:{discovery-port}`

the framework chooses what to specify for discovery port. `my-task.my-framework...` is the name of the A record that we generate. that A record either points to a container-ip or host-ip, but SRV record generation shouldn't care.

if there's no discovery info, then we generate:

- `_my-task._tcp.my-framework.mesos` --> `{slave-host}:{host-port}`